### PR TITLE
Travis tests and codecov clean

### PR DESCRIPTION
**Pull request by probably-not-a-cat**
This PR contains this PR https://github.com/AdventureLookup/adventurelookup-backend/pull/14

If this one is good, then we can just close the other one and merge in this one.

This has the basic starting models, tests running on travis, and 100% code coverage.

### CODE REVIEW

File: web/adventures/models.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+def parse_URL_list_field(url_list, separator):
+    """Takes a string of urls and coverts them into a list."""
+    if url_list == "":
+        return []
+    return url_list.split(",")
+
+class URLListField(models.URLField):
```
Comment (probably-not-a-cat) [inline web/adventures/models.py]: This may make more sense to say `class URLListField(models.Field):` instead. Thoughts?

File: .travis.yml [hunk 2]
```diff
@@ -18,5 +18,14 @@ before_install:
   - sudo apt-get update
   - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
 
+install:
+  - docker-compose run web pip install -r requirements.txt
```
Comment (pejter) [inline .travis.yml]: This is [done during image build](https://github.com/AdventureLookup/adventurelookup-backend/blob/master/web/Dockerfile#L16).
I would suggest removing it from here and maybe putting the build step into install as it is a part of the setup process and not running tests themselves.

File: web/adventures/models.py (hunk 1 above)
Comment (pejter) [inline web/adventures/models.py]: I would suggest inheriting from `models.CharField` instead, as this is a text field.

**Comment by pejter** [comment]
The `URLListFIeld` definition should be in `fields.py` to conform with the framework design and separate the models from the fields.

File: web/adventures/models.py (hunk 1 above)
Comment (probably-not-a-cat) [inline web/adventures/models.py]: Hmm, I was thinking that too, but I'm not sure that would make sense. It's essentially stored as a text field, but I expect people to interact with it as a list of strings. I wouldn't say that a list of strings is a subtype of a string, so does it make sense to say a list of URLs is a subtype of `CharField`?

**Pull request merged**